src/test/cli-integration/rbd/gwcli_client.t

```diff
@@ -0,0 +1,33 @@
+Login to the target
+===================
+  $ IP=`cat /etc/ceph/ceph.conf |grep 'mon host' | awk -F: '{print $2}'`
+  > iscsiadm -m discovery -t st -p $IP -l 2&> /dev/null
+  $ ls /dev/disk/by-path/ |grep 'iscsi-iqn.2003-01.com.redhat.iscsi-gw:ceph-gw' |wc -l
+  2
+
+Make filesystem
+===============
+  $ device=`multipath -l | grep 3600140536001405da17b74481464e9fa | awk '{print $1}'`
```

Comment:
Nit: need to find a way to make this generic since I would expect different multipath WWIDs

Reply:
Yeah, I had changed it to `LIO-ORG,TCMU device`, might be forgoten to amend it.